captum/optim/_utils/image/common.py

```diff
@@ -23,6 +79,37 @@ def get_neuron_pos(
     return _x, _y
 
 
+def _dot_cossim(
+    x: torch.Tensor,
+    y: torch.Tensor,
+    cossim_pow: float = 0.0,
+    dim: int = 1,
+    eps: float = 1.0e-4,
```

Comment:
It looks like the `eps` variable isn't being used anywhere in this function. Shouldn't it be being used for the dot cosine similarity calculations somewhere?

Reply:
Thanks for the catch! Yeah it should be passed to torch.cosine_similarity.